build/Tests/UnitTests/Foundation.WindowsOnly/Foundation.WindowsOnly.UnitTests.vcxproj

```diff
@@ -234,8 +234,6 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\Frameworks\Foundation\NSObject_NSCoding.mm" />
```

Comment:
Does the /wholearchive fix also go and remove some of these artifacts?

---
In reply to: [73436897](https://github.com/Microsoft/WinObjC/pull/806#discussion_r73436897) [](ancestors = 73436897,73436698)